### DISCUSSION TITLE
Integrating keycloak-config-cli?

### DISCUSSION THREAD

**Discussion by tanadeau** [keycloak/keycloak#23945] [Ideas]
The [keycloak-config-cli](https://github.com/adorsys/keycloak-config-cli) is an invaluable tool for administering Keycloak especially in local dev environments and Kubernetes infrastructure. For the latter, it's already used as part of the [Bitnami Keycloak Helm chart](https://artifacthub.io/packages/helm/bitnami/keycloak). It would be very useful if this was a first-party tool that had simultaneous release, QA, etc. with the actual application. From what I can tell, the current maintainers may be open to this.

**Comment by Wareload** [comment]
I would also like to see the keycloak config cli in the described spot and I am also using it at the moment.
Appearently, the keycloak terraform provider (https://github.com/keycloak/terraform-provider-keycloak) was migrated to keycloak, so now I'm between continuing using the third party keycloak config cli or using the official terraform provider.
Are there any informations on how to continue/recommendations or if there is interest in including the keycloak config cli as well like the terraform provider?
